src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedCasesService.kt

```diff
@@ -32,6 +32,11 @@ class UnallocatedCasesService(
     return contacts.minByOrNull { c -> c.contactStart }?.contactStart?.toLocalDate()
   }
 
+  fun getOffenderName(crn: String): String {
+    val offenderSummary = communityApiClient.getOffenderSummary(crn)
```

Comment:
I assume  there is no chance offendersummary is going to be empty/null ?

Reply:
I believe so, yeah. I queried dev with the known crn and it always returned something

Reply:
The only way this could happen (beyond a mistake on our part in passing the CRN around!) will be if we get an allocation demand event from a CRN as part of the process by which it is deleted. This is hard to imagine and is undercoverable anyway